### PR TITLE
Feature/ssl and redirect

### PR DESCRIPTION
**Pull request by berkes**
Fix '*74 no "ssl_certificate" is defined in server' error.

Error caused in unicorn nginx.conf. When ssl was on and site had redirect-domains. 

This also adds ssl-support for redirect-domains in passenger: allowing https versions to be redirected as well.

### CODE REVIEW

**Comment by jvanbaarsen** [comment]
@berkes Thanks! Can you also add an entry to the CHANGELOG?

**Comment by berkes** [comment]
Added entries to CHANGELOG. Had to rebase and force-push for that (sorry).

**Comment by jvanbaarsen** [comment]
No problem :-) I'll be on vacation next week, will test this when I get back on Tuesday!

**Comment by berkes** [comment]
Anything needed from me to take this forward?

**Comment by jvanbaarsen** [comment]
@berkes I don't think anything from your side is required. Its mostly a massive time constraint on my end. Maybe @michiels can also look at this PR so we can speed things up a little?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ This project makes use of the [Sementic Versioning](http://semver.org/)
 
 ### Added
 - Set the max db `pool` size via an ENV var called `DB_POOL_SIZE`
+- Fixed bug when redirect-domains were set on a site with SSL.
```
Comment (jvanbaarsen) [inline CHANGELOG.md]: Can you put this under a heading called `### Fixed`? (That way its easier for people that ran into this issue to spot if an issue has been solved for them :) )

**Comment by jvanbaarsen** [comment]
Looks good to me. @michiels Any thoughts?

**Comment by michiels** [comment]
Looks good! Happy to merge this in to also support the SSL for the redirect domains :)

**Comment by michiels** [comment]
Ah, only Jeroen's changelog comment to correct and then let's push the merge button!

**Pull request closed without merging**